packages/reactivity/src/baseHandlers.ts

```diff
@@ -25,7 +25,9 @@ import {
 import { isRef } from './ref'
 import { warn } from './warning'
 
-const isNonTrackableKeys = /*@__PURE__*/ makeMap(`__proto__,__v_isRef,__isVue`)
+const isNonTrackableKeys = /*@__PURE__*/ makeMap(
+  `__proto__,__v_isRef,__isVue,dep`,
+)
```

Comment:
This would break reactivity for a reactive property called `dep`:

- [Playground](https://deploy-preview-12804--vue-sfc-playground.netlify.app/#eNp9kU1PwzAMhv+KlQugTa0mOE3dxId2gAMgPm65lMzrsrVJlDhjUpX/TpKysQPazX7fx9ZruWd3xhQ7j2zKKiesNAQOyZs5V7Iz2hL0YLEWJHcIAVZWd3AR+QuuuBJaOYLONTA7Qpc9LNFMYQLhiquqHJbGdbEh7ExbE8YOoFpP5n2fpos4ACFUZVSy8+WJtIJb0UqxnXH2y4xGnGUA4NNksBzIKFblyXI2ZuRiuJVsio3TKh7XJ54zoTsjW7QvhmQMz9kUspO8um3191PWyHocH3SxRrH9R9+4fdI4e7Xo0O6Qs6NHtW2QBnvx/oz7WB/NTi99G+kz5hs63fqUccDuvVrG2CdcTvuYXyRV8+EWe0LlDkeloIkMmecsfuzhzOl/ca+LmzzHVWDhB7iIswM=)

Reply:
Agree with @skirtles-code.
Perhaps we can add the `__v_skip` property to the `Dep` class to prevent it from being tracked.

Reply:
Thanks for pointing this out! Push changed.